lexicons/app/bsky/feed/generator.json

```diff
@@ -0,0 +1,29 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.generator",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "A declaration of the existence of a feed generator",
+      "key": "named",
+      "record": {
+        "type": "object",
+        "required": ["hostname", "createdAt"],
+        "properties": {
+          "hostname": {"type": "string"},
```

Comment:
preference for the format here?

Do we include scheme? allow for port numbers? allow for additional pathing?

I think no on ports/pathing. maybe scheme?

- `feed.dholms.xyz`
- `https://feed.dholms.xyz`
- `http://feed.dholms.xyz:3000`
- `https://feed.dholms.xyz/api`

Reply:
IMO the record should have a broad definition, then the server can decide if it's supports the given format or not

Reply:
definitely no on pathing, but allowing ports might be nice. we could do "origin" and then override the scheme if we need to

Reply:
Truly don't want to get too big-brained about it, but I do wonder if referencing a service endpoint in the user's DID doc might not be a bad idea and answer some of these questions.  I'm just slightly uneasy about these endpoints changing via the repo without transparency and not being tied into identity at all.

Reply:
I actually really like this^^ gonna take another pass at it